README.md

```diff
@@ -1,13 +1,14 @@
-This library is used for triggering GPIO pins through a message.
-The usage for this would be for knowing that a publisher and a subscriber works on the same type of message - a kind of contract, you could say.
+# Gpio Messaging
```

Comment:
The title of the readme is now used as package name. It becomes "gpio_messaging".